js/app.js

```diff
@@ -0,0 +1,139 @@
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid r',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Thirth project',
+    title: 'Website Protfolio',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Sass', 'Codekit', 'Terminal'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita,',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Forth project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Sass', 'Bootstrap', 'JQuery'],
+    description:
+     'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita,',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Fifth project',
+    title: 'Website Protfolio',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['Terminal', 'Codepen', 'Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit. ',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Sixth project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['css', 'Codepen', 'Git & Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit.',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'last project',
+    title: 'Data Dashboard Healthcare',
+    image: './image_icons/Img_Placeholder.svg',
+    technologies: ['css', 'Codepen', 'Git & Github'],
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque porro aliquam quod repellat harum distinctio quas voluptates eligendi maiores at facilis asperiores, voluptatem placeat cum nostrum nisi quae, natus neque? Minus expedita, aliquid repellat recusandae laboriosam natus culpa odit.',
+    liveLink: '#',
+    sourceLink: '#',
+  },
+
+];
+
+const divProject = document.createElement('div');
+
+function addEvents() {
```

Comment:
- [ ] OPTIONAL
 The `See Project` button when clicked does not display a popup with details about the project. Please see as specified in the project **Portfolio: details popup window** [documentation guide](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Kindly endeavour to improve upon this aspect, and other outlined specifications in this guide